test/state/state.hpp

```diff
@@ -165,6 +178,9 @@ struct TransactionReceipt
     /// Amount of gas used by this transaction.
     int64_t gas_used = 0;
 
+    /// Amount of blob gas used by this transaction.
+    int64_t blob_gas_used = 0;
```

Comment:
This is always constant times number of blob hashes. Maybe replace this with a member `blob_gas_used() { GAS_PER_BLOB * blob_hashes.size() }`?

Reply:
We don't have access to `blob_hashes.size()` in the context of `TransactionReceipt`  struct.

Reply:
You should probably get this value form a transaction, not from receipt. This is not part of receipt as defined in EIP.

Reply:
OK. I get Pawel's comment now. Thanks.